patches/proton/keep-system-gstreamer-plugins.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/proton b/proton
+index ea6e9af..17ce908 100755
+--- a/proton
++++ b/proton
+@@ -607,7 +607,7 @@ class Session:
+ 
+         self.env["WINEDLLPATH"] = g_proton.lib64_dir + "/wine:" + g_proton.lib_dir + "/wine"
+ 
+-        self.env["GST_PLUGIN_SYSTEM_PATH_1_0"] = g_proton.lib64_dir + "gstreamer-1.0" + ":" + g_proton.lib_dir + "gstreamer-1.0"
++        self.env["GST_PLUGIN_PATH"] = g_proton.lib64_dir + "gstreamer-1.0" + ":" + g_proton.lib_dir + "gstreamer-1.0"
```

Comment:
This patch isn't needed since you're bundling gstreamer.

Reply:
Adressed.